src/BasketBundle/Controller/BasketController.php

```diff
@@ -35,6 +35,18 @@
  */
 class BasketController extends Controller
 {
+    /* @var sonata.basket.factory */
+    private $basket_factory;
+    
+    /**
+     * 
+     * @param type $basket_factory
+     */
+    public function __construct($basket_factory) 
```

Comment:
There is a trailing whitespace. If you run `make cs-fix`, you will get rid of it.